api/src/modules/movies/movies.handler.ts

```diff
@@ -21,6 +21,12 @@ export const handleGetMovies = async (request: Request, reply: Reply) => {
         where: { AND: [...filters] },
       }
     );
+    
+    if (movies.length === 0) {
+      reply.code(404);
+      throw new Error("No movies found");
```

Comment:
No movie found --> message to translate in french to throw in UI toast ?

Reply:
Fixed on [#fdbb1c6](https://github.com/Cinevoraces/cinevoraces/pull/41/commits/fdbb1c673f670326c67226ab4aa60a94878c67fb)